.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,48 @@
+# Work Item Issue Template
+description:
+title: "[Work_Item ]"
+labels: ["work item"]
+assignees: [shawnalpay, jpradocueva]
+## 1. **Problem Statement** *
+Describe the problem, issue, use case, or opportunity that this work item addresses.
+- **What is the problem?**: Explain the context and why it needs resolution.
+- **Impact**: Describe how the problem affects users, systems, or the project.
+
+Your input ...
+
+## 2. **Objective** *
+State the objective of this work item. What outcome is expected?
+- **Success Criteria**: Define how success will be measured (e.g. columns, KPIs, metrics, content).
+
+Your input ...
+
+## 3. **Supporting Documentation** *
+Include links to supporting documents such as:
+- Data Examples: [Link to data or relevant files]
+- Related Use Cases or Discussion Documents: [Link to discussion]
```

Comment:
We should have customer feedback including pull quotes.

Reply:
@timwright2000 Do you think it is sufficient to include feedback in Section 1 instead of here, per the suggestion I posted above, and per our discussion in the Member meeting?